### PR TITLE
Complete meta description and license files for both parsim and pyexp…

### PR DESCRIPTION
**Pull request by olawid**
Here are recipes for pyPI package "parsim", and its dependency "pyexpander". Both are pure python. 
I accept responsibility for maintaining these recipes.
@conda-forge/help-python. Thanks for your time...
/Ola

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by olawid** [comment]
I think this is ready to go now. I have limited both packages to Python >=3.5, as this is the requirement for parsim (pyexander recipe is included because parsim uses it).

File: recipes/pyexpander/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  imports:
+    - pyexpander
+
+about:
+  home: "https://goetzpf.bitbucket.io/pyexpander/index.html"
+  license: "GNU General Public v3 (GPLv3)"
+  license_family: "GPL3"
+  license_file: LICENSE.txt
+  summary: "a powerful macro processing language"
+  doc_url: ""
```
Comment (xhochy) [inline recipes/pyexpander/meta.yaml]: Please omit empty entries

File: recipes/pyexpander/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "27885ee926cb05bae3fb03e7572837b7233b41cd4c3f7bbc6cb9c099ccaefbcd"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (xhochy) [inline recipes/pyexpander/meta.yaml]: No need for these arguments anymore, they're there by default.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

File: recipes/parsim/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  number: 0
+  entry_points:
+    - psm=parsim.cli:psm_command
+    - psm_dakota_driver=parsim.dakota:psm_dakota_driver
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pandas
```
Comment (xhochy) [inline recipes/parsim/meta.yaml]: Are these requirements needed for building?

File: recipes/parsim/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+test:
+  imports:
+    - parsim
+  commands:
+    - psm --help
+
+about:
+  home: "https://gitlab.com/olwi/psm"
+  license: "GNU General Public v3 (GPLv3)"
```
Comment (xhochy) [inline recipes/parsim/meta.yaml]: Use SPDX specifiers
```suggestion
  license: GPL-3.0
```

File: recipes/parsim/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  license_family: "GPL3"
```
Comment (xhochy) [inline recipes/parsim/meta.yaml]:
```suggestion
  license_family: GPL
```

File: recipes/pyexpander/meta.yaml (hunk 1 above)
Comment (olawid) [reply recipes/pyexpander/meta.yaml]: Done

File: recipes/pyexpander/meta.yaml (hunk 2 above)
Comment (olawid) [reply recipes/pyexpander/meta.yaml]: Done

File: recipes/parsim/meta.yaml (hunk 3 above)
Comment (olawid) [reply recipes/parsim/meta.yaml]: Removed things not needed for building

File: recipes/parsim/meta.yaml (hunk 4 above)
Comment (olawid) [reply recipes/parsim/meta.yaml]: Done

File: recipes/parsim/meta.yaml (hunk 5 above)
Comment (olawid) [reply recipes/parsim/meta.yaml]: Done

**Pull request merged**